mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/bridge/VectorBridge.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.pojo.bridge;
+
+import org.hibernate.search.util.common.annotation.Incubating;
+
+/**
+ * An extension of a regular {@link ValueBridge} for vector fields.
+ */
+@Incubating
+public interface VectorBridge<V, F> extends ValueBridge<V, F> {
```

Comment:
This is a tagging interface... Not a fan :D 

But I'll see what it's used for...